src/cpp/desktop/DesktopMain.cpp

```diff
@@ -313,6 +313,12 @@ QString inferDefaultRenderingEngine()
       }
    }
 
+   if (::GetSystemMetrics(SM_REMOTESESSION))
```

Comment:
I think this check could happen before the enumeration of display devices (since we'll want to use software rendering regardless if running in remote desktop)

Reply:
Done!